web/libs/editor/src/components/Comments/Comment/CommentForm.scss

```diff
@@ -0,0 +1,110 @@
+.comment-form-enhanced {
+  $block: &;
```

Comment:
```
  $block: &;
```

What does this do?

Reply:
Ahh I found the usages later below. I don't really like this as it already felt hard to understand what the requirement was.

Reply:
can we just convert this to css module? then it won't require those playing with nesting

Reply:
@hlomzik How to do it in the current state of everything?